windows/benji.py

```diff
@@ -417,6 +418,67 @@ def main():
 			speak.say("Sorry Graph can not be Plotted")
 			speak.runAndWait()
 
+	#Box Office Status
+	elif link[-1] == "boxoffice":
+		try:
+			url = "https://in.bookmyshow.com/" + link[0] + "/movies/nowshowing"
+			r =  requests.get(url)
+			soup = BeautifulSoup(r.content, 'html.parser')
+
+			soup_level2 = []
+			show_status_list = []
+			shows_list = soup.find_all('div', attrs={'class': 'card-container wow fadeIn movie-card-container'})
+			for i in shows_list:
+				start = str(i).index("href=")
+				end = str(i).index("title=")
+				soup_level2.append("https://in.bookmyshow.com" + str(i)[start+6 : end-2])
+
+			show_status_raw = soup.find_all('div', attrs={'class': 'popularity sa-data-plugin'})
+			for i in show_status_raw:
+				start = str(i).index("data-coming-soon=")
+				end = str(i).index('data-event-code')
+				data = str(i)[start+18 : end-2]
+				
+				if data == "false":
+					show_status_list.append("In Cinemas Now...")
+				if data == "true":
+					show_status_list.append("Coming Soon...")
+
+			Tags_list = []
+			Name_list = []
+
+			for url in soup_level2:
+				r =  requests.get(url)
+				tags = BeautifulSoup(r.content, 'html.parser')
+				Tags_raw = tags.find_all('span', attrs={'class': '__genre-tag'})
+				tmp_tags = ""
+				for i in Tags_raw:
+					tmp_tags = tmp_tags + str(i)[str(i).index('">')+2 : str(i).index("</span>")] + " - "
+				Tags_list.append(tmp_tags[:-3])
+
+				Names_raw = tags.find_all('h1', attrs={'class': '__name'})
+				for i in Names_raw:
+					Name_list.append(str(i)[str(i).index('">')+2 : str(i).index("</h1>")])
+
+			speak.say("Preparing List")
+			speak.runAndWait()
+			cntr = len(Name_list)
+			print("----------------------------------------------")
+			print(link[0].capitalize())

```

Comment:
We should differentiate printing the names which are currently showing from the coming soon ones, something like
```
Print("Now showing")
Print(iterate through dict["Now showing"]
Print("Coming Soon")
Print(iterate through dict["Coming Soon"]
```